src/globus_sdk/services/auth/client/service_client.py

```diff
@@ -1222,47 +1219,38 @@ def create_client(
                 .. extdoclink:: Create Client
                     :ref: auth/reference/#create_client
         """
-        body: dict[str, t.Any] = {
-            "name": name,
-            "project": str(project),
-            "visibility": visibility,
-            "redirect_uris": redirect_uris,
-        }
-
-        if required_idp is not utils.MISSING:
-            body["required_idp"] = str(required_idp)
-        if preselect_idp is not utils.MISSING:
-            body["preselect_idp"] = str(preselect_idp)
-
         # Must specify exactly one of public_client or client_type
         if public_client is not utils.MISSING and client_type is not utils.MISSING:
             raise exc.GlobusSDKUsageError(
                 "AuthClient.create_client does not take both "
                 "'public_client' and 'client_type'. These are mutually exclusive."
             )
-
         if public_client is utils.MISSING and client_type is utils.MISSING:
             raise exc.GlobusSDKUsageError(
                 "AuthClient.create_client requires either 'public_client' or "
                 "'client_type'."
             )
-        if public_client is not utils.MISSING:
-            body["public_client"] = public_client
-        if client_type is not utils.MISSING:
-            body["client_type"] = client_type
 
+        body: dict[str, t.Any] = {
+            "name": name,
+            "project": project,
```

Comment:
Hang on. Why is this "project" when several other APIs here use "project_id"? Is this correct?